README.md

```diff
@@ -1,50 +1,56 @@
-# Mana
+# Mana Ethereum Implementation
```

Comment:
Is this the full name of the project?

Reply:
Not officially - I added to clarify what Mana is in the title of the repo but open to any other suggestions.

Reply:
there is a new trend in naming `https://github.com/paritytech/parity-ethereum` following go-ethereum 
could be `mana-ethereum`